CHANGES/8540.bugfix.rst

```diff
@@ -0,0 +1,7 @@
+WSS: Fix heartbeat timeout logic -- by :user:`arcivanov`.
+
+When a WebSocket pong message was not received, the
+ :py:meth:`~aiohttp.ClientWebSocketResponse.receive` operation did not terminate.
+ This change causes `_pong_not_received` to feed the `reader` an error message, causing
+ pending `receive` to terminate and return the error message. The error message contains
+ the exception :py:class:`~aiohttp.ServerTimeoutError`.
```

Comment:
I adjusted this to get the linter to pass. I tried to keep it as close to the original intent as possible. Please feel free to make additional changes if I missed something here.